config.go

```diff
@@ -0,0 +1,48 @@
+package wireless
+
+import (
+	"encoding/hex"
+
+	"github.com/pkg/errors"
+)
+
+// https://github.com/digsrc/wpa_supplicant/blob/515eb37dd1df3f4a05fc2a31c265db6358301988/src/utils/common.c#L631-L685
+func configParseString(s string) (string, error) {
```

Comment:
can you add a code comment on how this is different from the raw `DecodeSsid `?

Reply:
Added here: https://github.com/Plantiga/go-wireless/pull/2/commits/ae774a838736be5f46dd702704e1696e87a03dad